pytest/tests/mocknet/sharded_bm.py

```diff
@@ -402,33 +402,15 @@ def handle_get_profiles(args):
         logger.info(f"Targeting {machine}")
         args.host_filter = machine
 
-    if not args.skip_setup:
-        upload_args = copy.deepcopy(args)
-        script_dir = os.path.dirname(os.path.realpath(__file__))
-        upload_args.src = f"{script_dir}/helpers/get-profile.sh"
-        upload_args.dst = f"{REMOTE_HOME}/get-profile.sh"
-        run_remote_upload_file(CommandContext(upload_args))
-
     run_cmd_args = copy.deepcopy(args)
-    run_cmd_args.cmd = f"bash {REMOTE_HOME}/get-profile.sh {args.record_secs}"
+    run_cmd_args.cmd = f"bash {BENCHNET_DIR}/helpers/get-profile.sh {args.record_secs}"
     run_remote_cmd(CommandContext(run_cmd_args))
 
     os.makedirs(args.output_dir, exist_ok=True)
-    for host in CommandContext(args).get_targeted():
-        host_name = host.name()
-        logger.info(f"Downloading profile from {host_name}")
-        scp_cmd = [
-            "gcloud",
-            "compute",
-            "scp",
-            "--project=nearone-mocknet",
-            f"ubuntu@{host_name}:{REMOTE_HOME}/perf.script.gz",
-            f"{args.output_dir}/perf-{host_name}.gz",
-        ]
-        subprocess.run(
-            scp_cmd,
-            check=True,
-        )
+    download_args = copy.deepcopy(args)
+    download_args.src = f"{REMOTE_HOME}/perf*.script.gz"
```

Comment:
this `*` + adding the `$(hostname)` in the script seemed easier to me than trying to create a mini template language here.